modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -64,9 +64,12 @@ else()
         examples=0
         internal_only=0
         build_dir=${CMAKE_CURRENT_BINARY_DIR}
-        -j8
     )
 
+    if (DEFINED ARM_COMPUTE_SCONS_JOBS)
+        list(APPEND ARM_COMPUTE_OPTIONS -j${ARM_COMPUTE_SCONS_JOBS})
```

Comment:
May be if the option is not defined we should use:
https://cmake.org/cmake/help/latest/module/ProcessorCount.html

Reply:
When I tried to build natively on RaspberryPI maximum `-j` that passed was `-j2` but Raspberry have 4 cores. I suppose the issue is related to available RAM amount rather than core availability. Also if build would be triggered from OpenVINO there could be unfinished make jobs that reduce possible amount of simultaneous scons jobs even more.